jkube-kit/generator/webapp/src/test/java/org/eclipse/jkube/generator/webapp/handler/WildFlyAppSeverHandlerTest.java

```diff
@@ -25,74 +26,77 @@
 
 import mockit.Expectations;
 import mockit.Mocked;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
-public class WildFlyAppSeverHandlerTest {
+@SuppressWarnings({"unused", "ResultOfMethodCallIgnored"})
+class WildFlyAppSeverHandlerTest {
 
-  @Rule
-  public TemporaryFolder temporaryFolder = new TemporaryFolder();
+  @TempDir
+  Path temporaryFolder;
 
   @Mocked
   private GeneratorContext generatorContext;
 
   @Test
-  public void isApplicableHasJmsXmlShouldReturnTrue() throws IOException {
+  void isApplicableHasJmsXmlShouldReturnTrue() throws IOException {
     // Given
     // @formatter:off
     new Expectations() {{
-      generatorContext.getProject().getBuildDirectory(); result = temporaryFolder.getRoot();
+      generatorContext.getProject().getBuildDirectory(); result = temporaryFolder.toFile();
     }};
     // @formatter:on
-    assertTrue(new File(temporaryFolder.newFolder("META-INF"), "some-file-with-jms.xml").createNewFile());
-    assertTrue(new File(temporaryFolder.newFolder("META-INF-1337"), "context.xml").createNewFile());
+    Path webInf = Files.createDirectories(temporaryFolder.resolve("META-INF"));
```

Comment:
not webInf

Reply:
Does `metaInf` for `META-INF` work?

Please suggest a better name for `META-INF-1337`

Reply:
> Does `metaInf` for `META-INF` work?

Yes, of course.

> Please suggest a better name for `META-INF-1337`

Whatever you think is best.
Is this testing the behavior of an invalid META-INF directory? `invalidMetaInf` sounds good then.
I'm not in the best position to name the variables. The idea is that these variables describe what's actually going on and provide meaning to the test.